### DISCUSSION TITLE
Different RSSI readings in OSD and telemetry

### DISCUSSION THREAD

**Discussion by mazi14** [iNavFlight/inav#10427] [Support] [answered]
I have got a plane with SpeedyBee F405 mini wing FC, iNAV 7.1.1 and Walksnail V2 kit. During the flight I see almost all the time RSSI value just above 30 in my goggles (OSD), but when I looked in telemetry log file from my transmitter (Taranis X7), the common RSSI level is above 50. What could cuase this? Have you ever seen it? Another issue is "fuel consumtion" (current) measurement. Manual recommends ADC "current ratio" 195 for this FC and iNAV, but final value of "fuel consumption" is always higher than charged mAh for both my batteries. I land with the plane based on voltage. For one battery the difference is not such big, but for the second (voltage drop under load is bigger for this battery) the difference is significant. I tried to modify the "current ratio" value a bit, but without success. May be some offset will be needed. What is your experience with this FC?.

**Comment by MrD-RC** [comment]
Have you scaled the RSSI? FPort needs the scale to be set in INAV.

**Reply by mazi14** [reply]
I have not done this before. Yes, I am using FPort, but even when I have not scaled the RSSI (what value and where should I put it?) I would expect same readings in OSD and telemetry. Until I take off, telemetry RSSI is 100. RSSI value in OSD is definitely lower. Receiver should sent only one value (level of RSSI). So, where split of values happens? There was Matek F411 FC in this plane before and I have not noticed this difference until now.

**Reply by MrD-RC** [reply, marked as answer]
If you look on the setup page in configurator. The maximum RSSI is 50. This is what is sent from the receiver. You need to tell INAV this, or it will appear as half the actual RSSI. 

`set rssi_max = 50`

**Comment by mazi14** [comment]
Finally I have got it! Telemetry-receiver sends the RSSI over the air directly to the transmitter. OSD-FC gets RSSI via FPort from receiver, right?
